debug_toolbar/management/commands/debugsqlshell.py

```diff
@@ -2,12 +2,17 @@
 
 import sqlparse
 from django.core.management.commands.shell import Command  # noqa
-from django.db.backends import utils as db_backends_utils
+from django.db import connection
+
+if connection.vendor == "postgresql":
+    from django.db.backends.postgresql import base as base_module
+else:
+    from django.db.backends import utils as base_module
```

Comment:
Why is the postgresql branch necessary?

Reply:
Django has a new subclass for the CursorDebugWrapper for postgres. With Django 3 and postgres, the current version of the toolbar's `debug_shell` command doesn't work.